changelog.txt

```diff
@@ -1,9 +1,31 @@
+4.30.0
+==========
+### New Features:
+**Added support for Proof Key for Code Exchange (PKCE) in confidential client authorization code flows**. See issue [#1473](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/1473).  
+
+### Enhancements:
```

Comment:
i would think these would be bugs...what's the difference between an enhancement and a feature?

Reply:
Not a bug because it doesn't fix anything that's broken. Enhancements - minor changes to an existing feature, New Feature is a big code feature. I can just probably merge with New Feature section.

Reply:
minor changes to an existing feature is a bug, so i would just put it in bug category.